docs/packing.rst

```diff
@@ -179,17 +179,19 @@ When reproducing an experiment that communicates with a server, the experiment w
 
 and use *reprozip* to trace the script execution, rather than the experiment itself. In this way, ReproZip is able to capture the local server as well, which ensures that the server will be alive at the time of the reproduction.
 
-For example, to start postgresql, then a webserver (that will run until Ctrl+C is received), then stop postgresql::
+For example, if you have an web app that uses PostgreSQL and that runs until ``Ctrl+C`` is received, you can use the following script::
 
     #!/bin/sh
 
-    /etc/init.d/postgresql start    # Start PostgreSQL
+    /etc/init.d/postgresql start        # Start PostgreSQL
 
-    trap ' ' INT                    # Don't exit the whole script on Ctrl+C
+    trap ' ' INT                        
```

Comment:
I put it on the explanation, but feel free to add it back there (but leave
the one in the text).

On Friday, May 20, 2016, Remi Rampin <notifications@github.com> wrote:

> In docs/packing.rst
> <https://github.com/ViDA-NYU/reprozip/pull/187#discussion_r64089057>:
>
> >
> > -    trap ' ' INT                    # Don't exit the whole script on Ctrl+C
> > +    trap ' ' INT
>
> This command is the reason I'm showing the full script, shouldn't we keep
> the comment?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/reprozip/pull/187/files/5ee851567ecaf23af856b96c164aba781371642a#r64089057>
>


-- 
Sent from Gmail Mobile